lang/attribute/event/src/lib.rs

```diff
@@ -81,6 +81,35 @@ pub fn emit(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     })
 }
 
+#[proc_macro]
+pub fn emit_cpi(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
+    let tuple = parse_macro_input!(input as syn::ExprTuple);
```

Comment:
It would be better if we took 2 arguments as input instead of a tuple here.

Reply:
updated! Thanks